www/front_src/src/Resources/Details/detailsAtoms.ts

```diff
@@ -39,22 +34,21 @@ export const defaultSelectedCustomTimePeriodAtom = atom<
 
 export const selectResourceDerivedAtom = atom(
   null,
-  (_, set, resource: Resource) => {
+  (get, set, resource: Resource) => {
     set(openDetailsTabIdAtom, detailsTabId);
     set(selectedResourceUuidAtom, resource.uuid);
-    set(selectedResourceIdAtom, resource.id);
-    set(selectedResourceTypeAtom, resource.type);
-    set(selectedResourceParentTypeAtom, resource.parent?.type);
-    set(selectedResourceParentIdAtom, resource.parent?.id);
+    set(selectedResourcesDetailsAtom, {
```

Comment:
Why don't we also include uuid in resource details?

Reply:
i don't need uuid in storage either for  building endPoint for redirect  or  for panel close , it is used just in link parameters as information ,
http://localhost:4000/centreon/monitoring/resources?details={"customTimePeriod":{"end":"2022-07-28T15:42:01.540Z","start":"2022-07-27T15:42:01.540Z"},"id":26,"parentId":14,"parentType":"host","resourcesDetailsEndpoint":"/centreon/api/latest/monitoring/resources/hosts/14/services/26","selectedTimePeriodId":"last_24_h","tab":"details","tabParameters":{},"**uuid":"h14-s26**"} ?   🤔 🤔